Polyfills/Canvas/Source/Canvas.cpp

```diff
@@ -182,9 +180,9 @@ namespace Babylon::Polyfills
     Canvas::Impl& Canvas::Impl::GetFromJavaScript(Napi::Env env)
     {
         return *JsRuntime::NativeObject::GetFromJavaScript(env)
-            .Get(JS_CANVAS_NAME)
-            .As<Napi::External<Canvas::Impl>>()
-            .Data();
+                    .Get(JS_CANVAS_NAME)
+                    .As<Napi::External<Canvas::Impl>>()
+                    .Data();
```

Comment:
Is there an option to not do this?

Reply:
Since this is affected by ContinuationIndentWidth, setting it to 0 forces this to not indent. However, it would make other parts of the code where we want indentation to look wrong.